sycl/test-e2e/no_sycl_hpp_in_e2e_tests.cpp

```diff
@@ -7,7 +7,7 @@
 // CHECK-DAG: no_sycl_hpp_in_e2e_tests.cpp
 // CHECK-DAG: lit.cfg.py
 //
-// CHECK-NUM-MATCHES: 5
+// CHECK-NUM-MATCHES: 6
```

Comment:
What has caused this change? I don't see it used in the added test and in general we shouldn't increase this number.

Reply:
It's in the SYCL source string in the added E2E test:
https://github.com/intel/llvm/blob/407124bdda90e230347fa2d8f14cc932585293b0/sycl/test-e2e/KernelCompiler/kernel_compiler_sycl_jit.cpp#L31-L32